packages/components/pagination/src/page-size-selector/page-size-selector.tsx

```diff
@@ -8,7 +7,37 @@ import { warning } from '@commercetools-uikit/utils';
 import Label from '@commercetools-uikit/label';
 import messages from './messages';
 
-const mapRangeToListOfOptions = (perPageRange) => {
+export type PAGE_RANGE_SIZE = 's' | 'm' | 'l';
```

Comment:
```suggestion
export type TPageRangeSize = 's' | 'm' | 'l';
```

Reply:
Updated in 0930a7f385c5b35c8fe6f8120674d8f005ce7e8a

Thanks 👍